source/sessions/driver-sessions.rst

```diff
@@ -979,19 +977,19 @@ Test Plan
 =========
 
 The test plan SHOULD be run against both replica set and sharded cluster
-topologies.  It MUST NOT be run against a standalone server.
+topologies.
```

Comment:
Currently there is only one test file: dirty-session-errors.yml, which only runs on replica sets and sharded clusters.  Most of those tests will pass against a standalone (the ones that don't I think is because they use retryable writes).  

I request that we modify that test file (perhaps it will have to be split in two) so we can verify with tests that implicit and explicit sessions work against standalone servers.